docs/guides/1231-default-values-cue.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: /1231/default-values-cue
+displayed_sidebar: europa
+---
+
+# Default values and optional fields in Cue
```

Comment:
It's two topics, and I think it's already in the [CUE doc](https://docs.dagger.io/1215/what-is-cue/#default-values-and-the-nature-of-inheritance).
Why duplication?

Reply:
Smaller doc pages sound like a good idea to me. If this page is cohesive, it's OK to have duplication.